prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    const fullname = 'Timothy Buerger';
+    console.log(fullname);
+    const numberOfPets = '1';
```

Comment:
If we use quotes it makes it a string regardless of the content between the quotes. Here we want to assign `numberOfPets` an actual number `1`